test/e2e/play_kube_test.go

```diff
@@ -5331,4 +5331,23 @@ spec:
 		Expect(kube).Should(Exit(125))
 		Expect(kube.ErrorToString()).To(ContainSubstring("since Network Namespace set to host: invalid argument"))
 	})
+
+	FIt("podman kube down --force removes volumes", func() {
```

Comment:
```suggestion
	It("podman kube down --force removes volumes", func() {
```

Although I think you could just add this to an existing test which uses `subpathTestNamedVolume`